paulshryock/paul-shryock
# [#8]: Validate CSS

## Summary
- Validate CSS after building.

## Testing
- Run `npm start`
- See that CSS validation is logged to a JSON file (for now).

## Issue(s)

Closes #8

## Changelog

### Added
- Validate CSS. [#8]

## Release Checklist
